CHANGELOG.md

```diff
@@ -0,0 +1,26 @@
+# Change Log
```

Comment:
Can we back date this to all versions?

Reply:
> Can we back date this to all versions?

Possibly but it will take some investigation.